### PR TITLE
Android SDK 0.5.9: native email support

### PR DESCRIPTION
**Pull request by human-spider**
## Demo

http://void-docs.talkable.com/

### CODE REVIEW

File: source/android_sdk/upgrade.rst [hunk 1]
```diff
@@ -6,6 +6,11 @@ Upgrade
 
 Sometimes we need you to upgrade existing integration to use our latest features.
 
+0.5.9
+-----
+
+Added support for native email sharing.
```
Comment (dgtv) [inline source/android_sdk/upgrade.rst]: You should also document new `shareOfferViaNativeMail` method. Lets place it under `Advanced` section near to `Sharing via Facebook`.

Comment (dgtv) [reply source/android_sdk/upgrade.rst]: And maybe we should extend `Native integration via API` section with step of using a new method before creating an offer share for case of email sharing.

Comment (human-spider) [reply source/android_sdk/upgrade.rst]: I don't think we should document it. This method is not intended for the enduser, even though it's public. It's called in response to a post message from webview. The user won't be able to obtain an instance of `TalkableOfferFragment` to call it. Other similar methods such as `shareOfferViaSms` and `shareOfferViaFacebookMessage` are not documented.

Comment (dgtv) [reply source/android_sdk/upgrade.rst]: Missed that it is a method from the fragment. Ok than

**Pull request merged**